lib/openstudio/extension.rb

```diff
@@ -41,6 +41,24 @@ module Extension
     class Extension
       attr_accessor :root_dir
 
+      # max number of datapoints to run
+      MAX_DATAPOINTS = Float::INFINITY
```

Comment:
@TMaile MAX_DATAPOINTS, NUM_PARALLEL, and DO_SIMULATIONS should be used in [run_osws](https://github.com/NREL/openstudio-extension-gem/blob/develop/lib/openstudio/extension/runner.rb#L625) right?